client/blocks/editor-checkout-modal/index.tsx

```diff
@@ -2,7 +2,7 @@
  * External dependencies
  */
 import React, { useMemo, useEffect } from 'react';
-import { connect } from 'react-redux';
+import { connect, useSelector } from 'react-redux';
```

Comment:
Let's use just one way of using `react-redux`: `connect` HOC or the hooks. The component is already functional and removing `connect` should be very easy.

Reply:
Good point, addressed in https://github.com/Automattic/wp-calypso/pull/53184/commits/b54e7b66ab4fcaed94f82616b599ec9d0760ee90